oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticConnection.java

```diff
@@ -108,11 +121,25 @@ public RestHighLevelClient getClient() {
                         Header[] headers = new Header[]{new BasicHeader("Authorization", "ApiKey " + apiKeyAuth)};
                         builder.setDefaultHeaders(headers);
                     }
-                    client = new RestHighLevelClient(builder);
+                    hlClient = new RestHighLevelClient(builder);
+                    ElasticsearchTransport transport = new RestClientTransport(
+            	        hlClient.getLowLevelClient(),
+            	        new JacksonJsonpMapper()
+            	    );
+            	    esClient = new ElasticsearchClient(transport);
                 }
             }
         }
-        return client;
+        return hlClient;
+    }
+
+    /**
+     * Gets the NEW Elasticsearch Client
+     * @return the new Elasticsearch client
+     */
+    public ElasticsearchClient getElasticsearchClient() {
+    	getClient();
+	    return esClient;
```

Comment:
It looks like a tab character here... In our code (well, my code at least), we use spaces instead of tabs (4 spaces). Could you configure your IDE to use spaces instead of tabs? I know it's not the Eclipse default...